### ISSUE TITLE
Share login fail return sessionId when username contains 'dot'

### ISSUE THREAD

**Issue by MIRK0** [tynbendad/simplecgmanalog#8]
Hi. I noticed that Dexcom share LoginPublisherAccountByName fails when username contains non-alpha chars like a 'dot'. It always return a 00000000-0000-0000-0000-000000000000 sessionId, without checking password validity.
Are you aware of any method to escape username (I tried all possible combinations of backslash, underscore, % chars) to make it accept non-alpha chars?

**Comment by tynbendad** [comment]
No, i'm not aware - I haven't tried using non-alpha characters at all, but I have read about others having to change their usernames (i.e., create new dexcom accounts) and/or passwords because of issues with it.
